ambry-api/src/main/java/com/github/ambry/rest/RestUtils.java

```diff
@@ -281,6 +284,14 @@
     }
   }
 
+  public static final class QuotaHeaders {
+
+    /**
+     * Response header indicating cost incurred by the request against capacity unit and storage quotas.
+     */
+    public static final String REQUEST_COST = "x-ambry-request-cost";
```

Comment:
It's not a separate cost for read and write?  I created them as separate in the avro schema based on that understanding.

Reply:
The value of the header will contain the separate costs for read, write and storage. Something like below:
"READ_CAPACITY_UNIT=1.9; STORAGE_IN_GB=3.0; "